examples/bert/bert_train.py

```diff
@@ -420,8 +252,8 @@ def main(_):
         )
         optimizer = keras.optimizers.Adam(learning_rate=learning_rate_schedule)
 
-        pretraining_model = BertPretrainer(model)
-        pretraining_model.compile(
+        language_model = BertLanguageModel(encoder)
```

Comment:
Calling this a language models feels slightly off to me. I am used to seeing language model refer specifically to a transformer with a masked lm head or a causal lm head.

This is a model with a masked lm head AND a next sentence prediction head. I would call this the pretraining_model, or something like that.